frontend/src/app/components/cookies/cookies.component.html

```diff
@@ -0,0 +1,8 @@
+<div *ngIf="show" class="cookies">
+  <app-container>
+    <div class="cookies__content">
+      <p>Tato stránka využívá soubory cookies v rámci ověření totožností systémem Recaptcha.</p>
```

Comment:
"v rámci" -> "za účelem ověření, že nejste robot 🤖." 😄  prijde mi to vtipne

Reply:
emoji bych tam nedávala :D ještě vyděsíme nějakou babču